progressReport/canvasEmbedding.js

```diff
@@ -0,0 +1,52 @@
+const currentUrl = window.location.href;
+
+
+
+const targetPageUrl = 'https://ucberkeleysandbox.instructure.com/courses/141/pages/concept-map?module_item_id=418';
+const alternateTargetPageUrl = 'https://ucberkeleysandbox.instructure.com/courses/141/pages/concept-map';
+// Paste Canvas API key here
+const accessToken = '';
+const courseId = '141';
+const studentId = ENV.current_user_id;    // Replace with the actual student ID
```

Comment:
How are these going to be sourced?

Reply:
The `courseId` will be entered as a constant

I meant to remove the in-line comment on line 10. Sorry about that! Please disregard the comment. The `studentId` can be accessed by by `ENV.current_user_id`, which is the user id of the user for whom the page is being generated.